docs/vsphere-prerequisites.md

```diff
@@ -1,71 +1,34 @@
-# vSphere pre-requisites
+# vSphere Platform Pre-Requisites
 
-The vSphere cluster must be configured with [hybrid OVN Kubernetes networking](setup-hybrid-OVNKubernetes-cluster.md)
-with a custom VXLAN port to work around the pod-to-pod connectivity between
-hosts [issue](https://docs.microsoft.com/en-us/virtualization/windowscontainers/kubernetes/common-problems#pod-to-pod-connectivity-between-hosts-is-broken-on-my-kubernetes-cluster-running-on-vsphere).
+In order to successfully use the Windows Machine Config Operator (WMCO) on a vSphere Platform, 
+the following pre-requisites are required:
 
-In order to use the Windows MachineConfig Operator to successfully create
-a Windows node, you will first need to create a Windows "golden image"
-VM. These steps are an overview of the required configuration of the
-golden image.
+* The vSphere cluster must be configured with [hybrid OVN Kubernetes networking with a custom VXLAN port](setup-hybrid-OVNKubernetes-cluster.md#vSphere)
+  to work around the pod-to-pod connectivity between hosts [issue](https://docs.microsoft.com/en-us/virtualization/windowscontainers/kubernetes/common-problems#pod-to-pod-connectivity-between-hosts-is-broken-on-my-kubernetes-cluster-running-on-vsphere)
+* Set up a compatible [Windows virtual machine golden image](vsphere-golden-image.md)
```

Comment:
I would again say `Windows Server Semi-Annual Channel (SAC): Windows Server 2004 VM golden image` just to reinforce it at the higher level.